DigixGlobal/governance-ui-components
# Feature/dgdg 278

This feature is to allow Proposers to update milestone funding and expected reward. 

**Test Plan**

- Create a Proposal with 2 milestones
-  Set fundings to 2 ETH or more
- Set expected Reward to 2 ETH or more
- Get this proposal approved and be voted successfully and claim the result
- Edit Funding button should be visible once voting result has been claimed
- Verify that changes on the funding is reflected on the Proposal Details page and Milestones Accordion
